getting-started/quickstart.md

```diff
@@ -16,7 +16,7 @@ Once you have Docker up and running, follow the next steps
 `docker-compose -f infrastructure-docker-compose.yml up -d`  
 `docker-compose -f application-docker-compose.yml up -d`
 
-Then point your browser to [http://localhost:3000](http://localhost:3000).
+Then point your browser to [http://localhost:8080](http://localhost:8080).
```

Comment:
@skvskv if you point to 8080 you will go to the gateway, if you are running the demo client UI it should run on 3000. am I wrong? Can you see the UI in 8080?

Reply:
8080 works for me, while 3000 absolutely doesn't (that's why I sought for another probable port)

Reply:
Could you show a screenshot including the port? I have a feeling we mean different things by 'demo ui'